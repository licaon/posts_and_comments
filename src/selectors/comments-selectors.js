export const getComments = state => state.CommentsReducer;
export const getCommentsState = state => state.CommentsStateReducer;
