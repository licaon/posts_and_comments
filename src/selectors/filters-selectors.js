export const getSearchFiltersValue = state => state.SearchReducer;
