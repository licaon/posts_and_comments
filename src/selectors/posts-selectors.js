import { createSelector } from 'reselect';

export const getPosts = state => state.PostsReducer.posts;

export const getFilteredPosts = createSelector(
  [getPosts],
  posts => posts,
);
