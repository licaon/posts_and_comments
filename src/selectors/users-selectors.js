export const getUsers = state => state.UsersReducer.users;
