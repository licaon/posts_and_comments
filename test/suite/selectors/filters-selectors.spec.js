import * as FiltersSelectors from 'src/selectors/filters-selectors';

describe('FiltersSelectors', () => {
  it(`should execute getSearchFiltersValue correctly`, (done) => {
    expect(FiltersSelectors.getSearchFiltersValue({
      FilterSearchReducer: 'FilterSearchReducer',
    })).toMatchSnapshot();

    done();
  });
  it(`should execute getSelectFiltersValue correctly`, (done) => {
    expect(FiltersSelectors.getSelectFiltersValue({
      FilterSelectReducer: 'FilterSelectReducer',
    })).toMatchSnapshot();

    done();
  });
});
