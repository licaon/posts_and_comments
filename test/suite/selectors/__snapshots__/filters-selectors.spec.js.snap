// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FiltersSelectors should execute getSearchFiltersValue correctly 1`] = `"FilterSearchReducer"`;

exports[`FiltersSelectors should execute getSelectFiltersValue correctly 1`] = `"FilterSelectReducer"`;
